### PR TITLE
Updated Enhancement/1123 Tweak mobile styles for profile/Dojo profiles to include picture

### PR DESCRIPTION
**Pull request by BSKERRITT**
https://github.com/CoderDojo/community-platform/issues/1123

### CODE REVIEW

File: web/public/templates/dojos/dojo-detail.dust [hunk 1]
```diff
@@ -25,11 +25,13 @@
               </div>
             </div>
             <div cd-dojo-actions can-manage-dojo="isDojoAdmin" can-manage-events="isTicketingAdmin" dojo="dojo"></div>
-            <div cd-rounded-image src="https://s3-eu-west-1.amazonaws.com/zen-dojo-images/" src-id="{{ dojo.id }}" src-fallback="/img/avatars/dojo-default-logo.png" alt="{{ dojo.name }}" class="col-md-3 hidden-xs"></div>
-            <div class="col-md-6 cd-dojo-detail__dojo-name">
-              <h2 ng-bind="::dojo.name"></h2>
-              <h5 ng-if="dojo.website" class="cd-dojo-detail__dojo-website"><a ng-href="{{ dojo.website }}" title="{@i18n key="Website"/}"> <i class="fa fa-globe"></i> <span ng-bind="::dojo.website"></span></a></h5>
-            </div>
+              <div class="cd-profile__avatar">
+                <div cd-rounded-image src="{{ '/api/2.0/profiles/' + profile.id + '/avatar_img' }}" alt="{@i18n key="Avatar"/}" class="cd-profile__avatar"></div>
```
Comment (Wardormeur) [inline web/public/templates/dojos/dojo-detail.dust]: src differs as well as classes

File: web/public/templates/dojos/dojo-form.dust [hunk 2]
```diff
@@ -6,18 +6,19 @@
...
-                <label for="dojoName">
-                  <span>{@i18n key="Dojo Name"/}*</span>
-                </label>
-                <input type="text" ng-bind="::dojo.name" class="form-control h2" form="createDojoForm"
-                  required required-message="'{@i18n key="Name is Empty"/}'" ng-model="dojo.name" name="dojoName" id="dojoName"
-                  placeholder="{@i18n key="e.g. Docklands, Dublin 1 @ CHQ Building"/}" ng-change="updateLocalStorage('dojoListing','name', dojo.name)"
-                  uib-popover="{@i18n key="Please type Dojo name in English/Latin character a-Z!"/} {@i18n key="Normally named after an area, like 'Cork', or if there are multiple Dojos are in your area 'San Fran @ GitHub'. Try refrain from prefixing the name with 'CoderDojo'."/}"
-                  popover-trigger="focus" popover-placement="bottom">
+            <div class="cd-profile__header">
+              <div cd-rounded-image src="{{ '/api/2.0/profiles/' + profile.id + '/avatar_img' }}" alt="{@i18n key="Avatar"/}"
```
Comment (Wardormeur) [inline web/public/templates/dojos/dojo-form.dust]: same comment than ^ :)

File: web/public/templates/dojos/dojo-detail.dust (hunk 1 above)
Comment (DanielBrierton) [inline web/public/templates/dojos/dojo-detail.dust]: To clarify, the change to the class is fine. Just need to use the original `src` and `alt`, and include the `src-id` and `src-fallback` from the original code too.

File: web/public/templates/dojos/dojo-detail.dust [hunk 3]
```diff
@@ -25,11 +25,14 @@
...
             </div>
             <div cd-dojo-actions can-manage-dojo="isDojoAdmin" can-manage-events="isTicketingAdmin" dojo="dojo"></div>
-            <div cd-rounded-image src="https://s3-eu-west-1.amazonaws.com/zen-dojo-images/" src-id="{{ dojo.id }}" src-fallback="/img/avatars/dojo-default-logo.png" alt="{{ dojo.name }}" class="col-md-3 hidden-xs"></div>
-            <div class="col-md-6 cd-dojo-detail__dojo-name">
-              <h2 ng-bind="::dojo.name"></h2>
-              <h5 ng-if="dojo.website" class="cd-dojo-detail__dojo-website"><a ng-href="{{ dojo.website }}" title="{@i18n key="Website"/}"> <i class="fa fa-globe"></i> <span ng-bind="::dojo.website"></span></a></h5>
-            </div>
+              <div class="cd-profile__avatar">
+                <div cd-rounded-image src="https://s3-eu-west-1.amazonaws.com/zen-dojo-images/" src-id="{{ dojo.id }}" src-fallback="/img/avatars/dojo-default-logo.png" alt="{{ dojo.name }}"
+                alt="{@i18n key="Avatar"/}" class="cd-profile__avatar"></div>
```
Comment (Wardormeur) [inline web/public/templates/dojos/dojo-detail.dust]: duplicate alt

File: web/public/templates/dojos/dojo-form.dust [hunk 4]
```diff
@@ -6,18 +6,19 @@
...
-                <label for="dojoName">
-                  <span>{@i18n key="Dojo Name"/}*</span>
-                </label>
-                <input type="text" ng-bind="::dojo.name" class="form-control h2" form="createDojoForm"
-                  required required-message="'{@i18n key="Name is Empty"/}'" ng-model="dojo.name" name="dojoName" id="dojoName"
-                  placeholder="{@i18n key="e.g. Docklands, Dublin 1 @ CHQ Building"/}" ng-change="updateLocalStorage('dojoListing','name', dojo.name)"
-                  uib-popover="{@i18n key="Please type Dojo name in English/Latin character a-Z!"/} {@i18n key="Normally named after an area, like 'Cork', or if there are multiple Dojos are in your area 'San Fran @ GitHub'. Try refrain from prefixing the name with 'CoderDojo'."/}"
-                  popover-trigger="focus" popover-placement="bottom">
+            <div class="cd-profile__header">
+              <div cd-rounded-image src="{{ dojoImageUrl }}" alt="{@i18n key="Avatar"/}" src-id="{{ dojo.id }}" alt="{{ dojo.name }}"
```
Comment (Wardormeur) [inline web/public/templates/dojos/dojo-form.dust]: duplicate alt

**Pull request merged**